static/bib/yoshitom.bib

```diff
@@ -1,3 +1,30 @@
+@misc{yoshitom-levorato-2020-icpr,
+  author = {Yoshitomo Matsubara and Marco Levorato},
+  title = {Neural Compression and Filtering for Edge-assisted Real-time Object Detection in Challenged Networks},
+  year = {2020},
+  archiveprefix = {arXiv},
+  code = {https://github.com/yoshitomo-matsubara/hnd-ghnd-object-detectors},
+  eprint = {2007.15818},
+  primaryclass = {cs.CV},
+}
+
+@misc{yoshitom-levorato-2020-emdl,
+  author = {Yoshitomo Matsubara and Marco Levorato},
+  title = {Split Computing for Complex Object Detectors: Challenges and Preliminary Results},
+  year = {2020},
+  archiveprefix = {arXiv},
+  code = {https://github.com/yoshitomo-matsubara/hnd-ghnd-object-detectors},
+  eprint = {2007.13312},
+}
+
```

Comment:
These two entries should go to `static/bib/iasl.bib` as publications
done within the lab. Personal BibTeX files are only for papers done
before joining the lab or done on a side (i.e., outside of the lab’s
research line).